app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ResourcesController < ApplicationController
+  def index
+    category_id = params[:category_id]
+    @resources = if category_id
+                   Category.find(category_id).resources
```

Comment:
Yeah, no problem. There's a learning curve to `ActiveRecord`. It's not immediately clear what it ends up doing under the hood. It takes experience.